wifiphisher/pywifiphisher.py

```diff
@@ -595,6 +595,10 @@ def start(self):
                        "rogue Access Point").format(G, W, ap_iface.get_name())
 
             kill_interfering_procs()
+            #randomize the adapters MAC addresses
+            self.network_manager.randomize_interface_mac(ap_iface)
+            self.network_manager.randomize_interface_mac(mon_iface)
```

Comment:
If the user runs the tool with the `--nojamming` option (where now only one interface is used and ap_iface == mon_iface), the randomize function will run twice for a single interface.

Maybe it makes more sense to randomize each interface as soon as we assign it to either ap_iface or mon_iface.